static/src/stylesheets/head.content.scss

```diff
@@ -38,3 +38,4 @@
    ========================================================================== */
 
 @import 'module/atoms/_quiz';
+@import 'module/atoms/_youtube';
```

Comment:
@GHaberis where does our current video styling come from? It would be good to put it in the same place. It might work out well to have a comment explaining that it may need to be deleted once the youtube migration is complete (unless I have misunderstood and that isn't happening 😆 )